be/src/exprs/runtime_filter_bank.h

```diff
@@ -100,6 +100,7 @@ class RuntimeFilterBuildDescriptor {
     JoinRuntimeFilter* runtime_filter() { return _runtime_filter; }
     void set_is_pipeline(bool flag) { _is_pipeline = flag; }
     bool is_pipeline() const { return _is_pipeline; }
+    int8_t join_mode() const { return _join_mode; };
```

Comment:
can we use an enum as return type, which maybe more intuitive than int8_t?

Reply:
using int8_t is a legacy issue,  serde method pack it into byte string as a int8_t;
```
size_t JoinRuntimeFilter::max_serialized_size() const {
    // todo(yan): noted that it's not serialize compatible with 32-bit and 64-bit.
    auto num_partitions = _hash_partition_bf.size();
    size_t size = sizeof(_has_null) + sizeof(_size) + sizeof(num_partitions) + sizeof(_join_mode);
    if (num_partitions == 0) {
        size += _bf.max_serialized_size();
    } else {
        for (const auto& bf : _hash_partition_bf) {
            size += bf.max_serialized_size();
        }
    }   
    return size;
}
```